src/common/mfu_daos.c

```diff
@@ -218,10 +218,32 @@ static int daos_check_args(
         return 0;
     }
     
+    /* if passed in values for src/dst are both uuids ignore the case when comparing */
+
     /* Determine whether the source and destination
      * use the same pool and container */
-    bool same_pool = (strcmp(da->src_pool, da->dst_pool) == 0);
-    bool same_cont = same_pool && (strcmp(da->src_cont, da->dst_cont) == 0);
+    int rc1, rc2 = 0;
+    bool same_pool = false;
+    uuid_t src_pool;
+    uuid_t dst_pool;
+    rc1 = uuid_parse(da->src_pool, src_pool); 
+    rc2 = uuid_parse(da->dst_pool, dst_pool); 
+    if (rc1 == 0 && rc2 == 0) {
+        same_pool = (strcasecmp(da->src_pool, da->dst_pool) == 0);
```

Comment:
Suggestion: For this case, since we know they are both UUIDs, and we've already parsed them, maybe it's clearer to use uuid_compare? https://linux.die.net/man/3/uuid_compare
But I'm okay with either.